lib/copy-sync/__tests__/copy-sync-dir.test.js

```diff
@@ -27,15 +27,15 @@ describe('+ copySync()', () => {
       src = path.join(TEST_DIR, 'src')
       dest = path.join(TEST_DIR, 'dest')
 
-      fs.mkdirsSync(src)
+      fs.mkdirSync(src)
```

Comment:
> @jprichardson Perhaps we should formalize some collaborator guidelines sometime? :+1: 

> Why the conversion here to mkdirSync from mkdirsSync?

As @RyanZim mentioned, I thought we wouldn't need to call `mkdirs` since we know the parent directory exist. So, I guess I should revert to `mkdirsSync`. Is that right?

And also sorry that I am still not in harmony with you guys. I try my best to be as helpful as possible.